lib/Builder.css

```diff
@@ -173,17 +174,7 @@ html {
     }
 
     &__trigger {
-      width: 30px;
-      height: 30px;
-      padding: 2px;
-      border: 2px solid lightgray;
-      border-radius: 50%;
-      cursor: pointer;
-      transition: all 300ms ease;
-
-      &:hover {
-        transform: scale(1.15);
-      }
+      cursor: ew-resize;
     }
   }
 
```

Comment:
color-palette__button btn-add btn

padding: 0.7rem;
border-radius: 0.3rem;

Reply:
also let's add some margin between label and select:

.predefined-palette, .chart-types
margin-top: 0.5rem;

NOTE: class chart-types has a very generic name, it's possible that this name is duplicated in the dashboard builder - which would bring css conflicts

Reply:
.builder__header, .builder__header {
margin-top: 0

Reply:
let's remove border-radius from 
color-picker__trigger

let's remove font weight bold from
color-picker__label

let's change our h2 to
h2
font-weight: normal;
    font-size: 1.5rem;